index.html

```diff
@@ -64,5 +64,5 @@ <h3 id="statistics-header">
     <script src="js/statistics.js"></script>
     <script src="js/scripts.js"></script>
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/PapaParse/5.1.0/papaparse.min.js"></script>
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC2DiuOwQf-LdnOSNGW-mhlZt024RO0eL8&v=beta&libraries=marker,geometry&callback=initMap" async defer></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC2DiuOwQf-LdnOSNGW-mhlZt024RO0eL8&v=beta&libraries=marker&callback=initMap" async defer></script>
```

Comment:
What was geometry for?

Reply:
I added geometry for the last PR when I was calculating spherical distance between two sets of coordinates to retrieve the name of an institution which was pretty dumb, now it works by just getting it from the JS object.